tests/test_caching.py

```diff
@@ -670,12 +670,15 @@ def test_solve(self, operate_on_empty_cache):
         del eqn
         del grid
 
+        # Only deleted `u` shift reference still existing
         # `u` points to the various Dimensions, the Dimensions point to the various
-        # spacing symbols, hence, we need three sweeps to clear up the cache
-        assert len(_SymbolCache) == 12
+        # spacing symbols, hence, we need four sweeps to clear up the cache
```

Comment:
this comment deserves an extension for the extra sweep

Reply:
reminder

Reply:
fixed